### DISCUSSION TITLE
Issue in the notifications section

### DISCUSSION THREAD

**Discussion by orelvis15** [community/community#126741] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I was deleting notifications from my account and I noticed some strange behavior when trying to select notifications in the middle of a request.

https://github.com/community/community/assets/19788313/991f5869-12a0-4144-b951-25f8dcd2f0c4

**Comment by ebndev** [comment]
Hi @orelvis15, thanks for flagging this!

I am logging this as an issue since many users have had this problem. The GitHub product team may evaluate the feedback but there may not be an immediate resolution as they are not able to respond to every submission.

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
